### PR TITLE
Add a VU meter to the active mic selection

### PR DESCRIPTION
**Pull request by zzmp**

### CODE REVIEW

**Comment by zzmp** [comment]
### QA Plan

- Open the audio menu.
- Ensure the VU meter is on the active microphone.
- Change active microphones - make sure it stays accurate.

File: interface/resources/qml/hifi/audio/InputLevel.qml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        id: colors;
+
+        readonly property string muted: "#E2334D";
+        readonly property string gutter: "#575757";
+        readonly property string greenStart: "#39A38F";
+        readonly property string greenEnd: "#1FC6A6";
+        readonly property string red: colors.muted;
+    }
+
+    Text {
```
Comment (zfox23) [inline interface/resources/qml/hifi/audio/InputLevel.qml]: This is supposed to be `Text` and not `RalewayRegular` or something?

**Comment by YuppyPlays** [comment]
Will test when built :)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/926/HighFidelity-Beta-PR10762-0075d8619a7e443bee365ebb1ca1957b401a2564.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/926/HighFidelity-Beta-PR10762-0075d8619a7e443bee365ebb1ca1957b401a2564.dmg

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
@zzmp Just to clarify VU meter refers to the "Show Audio Level Meter" checkbox?

File: interface/resources/qml/hifi/audio/InputLevel.qml (hunk 1 above)
Comment (zzmp) [inline interface/resources/qml/hifi/audio/InputLevel.qml]: Yeah, to be consistent with the other mute text

**Comment by YuppyPlays** [comment]
Testing complete:

I can confirm the Audio Level Meter shows at the side of the selected audio input when un-muted.
image
image

**Pass** :)

**Pull request merged**